enh(cim): improve sharing dashboards (#4800)
centreon/centreon-documentation: versioned_docs/version-25.10/alerts-notifications/dashboards.md

1. Click the **share** button at the bottom right of the dashboard.
2. In the **Add contact** dropdown list, select the user to add.

>   > The visible contacts are those they respect these two conditions jointly;

Reviewer: ```suggestion
  > Only users meeting the following two conditions are displayed:
```

>   > - belong to the same **access groups** as yours,

Reviewer: ```suggestion
  > - belong to the same **access groups** as yours,
```

>   > - have access to the Dashboard menu (which can be activated via access rights).

Reviewer: ```suggestion
  > - have access to the Dashboards menu (can be activated using access rights).
```
